src/main/java/com/jcabi/email/Protocol.java

```diff
@@ -42,11 +42,13 @@
  * <pre> Postman postman = new Postman.Default(
  *   new SMTP(
  *     new Token("user", "password").access(
- *       new Protocol.SMTPS("smtp.gmail.com", 587)
+ *       new Protocol.SMTP("bind", "port")
```

Comment:
@mkordas I don't know what you mean - that is the only pair of bind/port google provides for sending email, and as I said above, it does not work with SMTP protocol through SMTP wire. If I leave it there, it's not a good example since it does not work. People will start looking into their code or simply assume there's a bug in our lib. 

This is a javadoc for SMTP, not for SMTPS.